srcpkgs/rust-analyzer/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'rust-analyzer'
+pkgname=rust-analyzer
+version=2020.07.13
+revision=1
+_ver=${version//./-}
+wrksrc="${pkgname}-${_ver}"
+build_style=cargo
+make_install_args="--path=crates/${pkgname}"
```

Comment:
can you explain why this is needed?

Reply:
Because the Cargo build style [sets it when nothing is set](https://github.com/void-linux/void-packages/blob/5474e8770623edc15f2c552e2beae77f86e729d8/common/build-style/cargo.sh#L20). If it the build style didn't set it, it would just work, but since it sets it to `.`, I have to override it, otherwise the package won't compile.